lib/plugin.js

```diff
@@ -27,10 +27,25 @@ module.exports = {
 
   printers: {
     "prisma-ast": {
-      print(path) {
+      print(path, options) {
         const { format } = require("@prisma/prisma-fmt-wasm");
         const { text } = path.getValue();
-        return format(text);
+
+        return format(
+          text,
+          JSON.stringify({
+            textDocument: {
+              content: text,
+              languageId: "prisma",
+              uri: options.filepath,
+              lineCount: text.split("\n").length,
+            },
```

Comment:
The rest of the object is something vscode specific that isn't in the protocol, IIRC

Reply:
I've tried a lot of combinations. I think that `serde` is failing to parse the JSON, and it's silently returning the unformatted input: https://github.com/prisma/prisma-engines/blob/34df67547cf5598f5a6cd3eb45f14ee70c3fb86f/prisma-fmt/src/lib.rs#L37-L43

Reply:
So I rebuilt `prisma-fmt-wasm` using `wasm-logger`, and found from the logs that it fails to parse the URI because of the wrong format, adding `file://` prefix helped, sorry for the disturbance 😅